source/comply/audit-log.rst

```diff
@@ -3,39 +3,205 @@ Audit Log v2 (Experimental) (E20)
+
+   
 **Accessing configuration options for audit log**
 
-Open ``config.json`` and navigate to the audit settings. The following `configuration options <https://docs.mattermost.com/configure/configuration-settings.html#audit-settings>`_ are available:
+Open ``config.json`` and navigate to the audit settings under ``ExperimentalAuditSettings``. Within the setting ``AdvancedLoggingConfig`` you are able to specifiy a filespec to another config file, a database DSN, or JSON. 
+
+The logging configuration JSON is an object (unordered collection) containing names and log target values. Each log target contains a type, options specific to the target type, format, and levels.
+
+The example below specifies one log target that outputs to the console using a plain text format with pipes delimiting fields. The standard log levels are listed, with only `error` and lower outputing a stack trace. The `error` level will output in red for log targets and formatters that support color output.
+
+.. code-block:: json
+
+  {
+      "sample-console": {
+        "type": "console",
+        "options": {
+          "out": "stdout"
+        },
+        "format": "plain",
+        "format_options": {
+          "delim": " | "
+        },
+        "levels": [
+          {"id": 5, "name": "debug"},
+          {"id": 4, "name": "info"},
+          {"id": 3, "name": "warn"},
+          {"id": 2, "name": "error", "stacktrace": true, "color": 31},
+          {"id": 1, "name": "fatal", "stacktrace": true},
+          {"id": 0, "name": "panic", "stacktrace": true}
+        ],
+        "maxqueuesize": 1000
+      }
+    }
+    
+.. Note::
+    Filenames for ``AdvancedLoggingConfig`` can contain an absolute filename, a relative filename, or embedded JSON.
+
+See the :download:`Advanced Logging Options Sample JSON ZIP file <../samples/advanced-logging-options-sample-json.zip>` for a sample configuration file.  
+
+Log Target Types
+~~~~~~~~~~~~~~~~
+Log target types include Console, Syslog, File, or TCP. 
+
+**Console configuration options:**
+
+.. csv-table::
+    :header: "Key", "Type", "Default", "Description"
+       
+       "out", "string", " ", "One of `stdout` or `stderr`. "
+
+**Syslog configuration options:**
+
+.. csv-table::
+    :header: "Key", "Type", "Default", "Description"
+       
+       "host", "string", " ", "IP or domain name of server to receive log records."
+       "port", "number", "", "Port number for server receiving log records."
+       "tls", "bool", "false", "When true, a TLS connection will be created."
+       "cert", "string", "  ", "Path to a cert file (.pem) to be used when establishing a TLS connection."
+       "insecure", "bool", "false", "When true, Focalbaord will accept any certificate presented by the server and any host name in that certificate. Should be used only in testing environments."
```

Comment:
```suggestion
       "insecure", "bool", "false", "When true, Boards will accept any certificate presented by the server and any host name in that certificate. Should be used only in testing environments."
```

Reply:
Are these the board docs or channels?

Reply:
@wiggin77 These are outside of channels - Audit log is under administration.  I see this more of a "platform" type feature, so wanted to remove it from the verticals specifically.

Reply:
Thanks @wiersgallak. If these docs are not product specific shouldn't we leave `Boards` product name out of these config docs?  The same docs apply to channels and boards.

Reply:
The accepted suggestion below works here:
```
"insecure", "bool", "false", "When true, any certificate will be accepted and any host name in that certificate. Should be used only in testing environments."
```